docs/manual/getting_started/benchmarking.rst

```diff
@@ -1,124 +1,76 @@
 Benchmarking Tools
 ==================
 
-Introduction
-------------
-
-|var-project-short| provides a tool that primarily measures data *throughput*
-and *latency* of the DDS applications within the network or
-within the same board, namely ``ddsperf``. This tool also helps 
-sanity checks to ensure your configuration is correctly set up and
-running. This chapter describes how to use the ``ddsperf`` tool and 
-read and interpret its outputs and results. Using the |var-project-short| Python
-package, you can also run ``ddsperf`` as a graphical application, by running
-``cyclonedds performance``.
-
-As well as ``ddsperf``, you can find dedicated examples in the
-product distribution that measure the DDS system throughput and the
-latency with their associated codebase. You can start from the provided
-code and customize it to fit your scenario and exact data types. Both the
-``ddsperf`` tool and the provided examples perform the benchmarking using
-sequences of octets with different parameterized sizes.
+The |var-project-short| ``ddsperf`` tool primarily measures data *throughput*
+and *latency* of the DDS applications within a network, or within the same board. 
+the ``ddsperf`` tool also helps to do sanity checks that ensure your configuration 
+is correctly set up and running. 
 
-.. important::
-
-    The ``throughput`` and ``latency`` examples are not suitable as general
-    DDS benchmarking tools since they sacrifice performance for being as simple
-    as possible. You cannot compare test results from these tools with results from other tools.
-
-Testing your network configuration
-----------------------------------
-
-Once your |var-project-short| installation is successfully completed, you may
-want to test if your network environment is correctly set up. This can
-be done by running the **Hello World!** example or using the
-``ddsperf`` tool. The **Hello World!** example sends a single message.
-In contrast, the ``ddsperf`` tool can send a continuous stream of data at a
-variable frequency for sanity checks and bypass sporadic network issues.
-
-The ``ddsperf`` tool is pre-installed within ``<installation-dir>/bin``.
-
-Complete the sanity checks of your DDS based system using the ``ddsperf``
-tool as follows:
-
-.. code-block:: bash
-
-    ddsperf sanity
-
-The sanity option sends only one data sample each second (1Hz).
-
-In another terminal, start the ``ddsperf`` with the **Pong** mode to echo
-the data to the first instance of the ``ddsperf`` started with the
-*Sanity* option.
-
-.. code-block:: bash
-
-    ddsperf pong
+For the |var-project-short| Python package, ``ddsperf`` provides a graphical application 

```

Comment:
I would rephrase this, as `ddsperf` does not provide anything graphical as far as I understand it. It is `cyclonedds performance` that provides a graphical wrapper around `ddsperf`.

Reply:
Removed.